evalml/model_understanding/graphs.py

```diff
@@ -690,14 +690,24 @@ def partial_dependence(
 
     _raise_value_error_if_mostly_one_value(feature_list, percentiles[1])
     wrapped = evalml.pipelines.components.utils.scikit_learn_wrapped_estimator(pipeline)
-    preds = sk_partial_dependence(
-        wrapped,
-        X=X,
-        features=features,
-        percentiles=percentiles,
-        grid_resolution=grid_resolution,
-        kind=kind,
-    )
+    try:
+        preds = sk_partial_dependence(
+            wrapped,
+            X=X,
+            features=features,
+            percentiles=percentiles,
+            grid_resolution=grid_resolution,
+            kind=kind,
+        )
+    except ValueError as e:
+        if "percentiles are too close to each other" in str(e):
+            raise ValueError(
+                "The scale of these features is too small and results in"
```

Comment:
Is there a way that users would be able to run PDP on this small data? ie change the percentiles? Or is it that if the data is too small/close together, partial dependence won't work at all?

If it's the former, I think it'd be nice if we can include a comment to give the user suggestions on how to change their params. Otherwise, this looks fine!

Reply:
I think you could artificially scale the feature by perhaps multiplying by 1E6 or something like that to defeat the absolute tolerance check.  That might fall within the realm of our current transformation components, but I think the story was just for a more informative error message.  Let me add a little bit to the error message.